webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -6,7 +6,7 @@ import { testConnection } from '../foreman_compute_resource';
 export function templateSelected(item) {
   const template = $(item).val();
 
-  if (!item.disabled) {
+  if (!item.disabled && template !== null) {
```

Comment:
imho you should early exit the function instead. also, i know the current code coverage is bad, but maybe consider adding a test? (should be as simple as passing an empty select as item to the function)

Reply:
an integration test? I don't think this currently has any other tests